packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -341,6 +341,8 @@ export class GroupedListSection extends BaseComponent<IGroupedListSectionProps,
       compact
     } = this.props;
 
+    const nestingDepth = (subGroup.level && subGroup.level + 1) || groupNestingDepth;
```

Comment:
NIT: Perhaps a ternary would be more readable.
```suggestion
    const nestingDepth = subGroup.level ? subGroup.level + 1 : groupNestingDepth;
```